src/transformers/modelcard.py

```diff
@@ -315,6 +315,8 @@ def _insert_values_as_list(metadata, name, values):
         return metadata
     if isinstance(values, str):
         values = [values]
+    if None in values:
+        values.remove(None)
```

Comment:
This will only remove the first `None` value, would it be better to do something like
```py
values = [v for v in values if v is not None]
# or
values = list(filter(None, values))
```
?

Reply:
Good point!